decidim-debates/decidim-debates.gemspec

```diff
@@ -6,17 +6,20 @@ require "decidim/debates/version"
 
 # Describe your gem and declare its dependencies:
 Gem::Specification.new do |s|
-  s.name = "decidim-debates"
-  s.summary = "Decidim debates module"
-  s.description = "A debates component for decidim's participatory spaces."
   s.version = Decidim::Debates.version
   s.authors = ["Josep Jaume Rey Peroy", "Marc Riera Casals", "Oriol Gual Oliva", "Genis Matutes Pujol"]
   s.email = ["josepjaume@gmail.com", "mrc2407@gmail.com", "oriolgual@gmail.com", "genis.matutes@gmail.com"]
+  s.license = "AGPL-3.0"
+  s.homepage = "https://github.com/decidim/decidim"
+  s.required_ruby_version = ">= 2.3"
 
+  s.name = "decidim-debates"
+  s.summary = "Decidim debates module"
+  s.description = "A debates component for decidim's participatory spaces."
   s.files = Dir["{app,config,db,lib}/**/*", "Rakefile", "README.md"]
 
-  s.add_dependency "decidim-comments"
-  s.add_dependency "decidim-core"
+  s.add_dependency "decidim-comments", Decidim::Debates.version
```

Comment:
Whoops, that was me when importing the code from the external module, thanks for noticing!